nativelib/src/main/scala/scala/scalanative/unsafe/package.scala

```diff
@@ -140,16 +140,20 @@ package object unsafe {
   /** Convert a CString to a String using given charset. */
   def fromCString(cstr: CString,
                   charset: Charset = Charset.defaultCharset()): String = {
-    val len   = libc.strlen(cstr).toInt
-    val bytes = new Array[Byte](len)
-
-    var c = 0
-    while (c < len) {
-      bytes(c) = !(cstr + c)
-      c += 1
+    if (cstr == null.asInstanceOf[CString]) {
+      null.asInstanceOf[String]
```

Comment:
I'm pretty sure the `asInstanceOf`s are not necessary:
```suggestion
    if (cstr == null) {
      null
```

Reply:
Thank you for the timely review & the suggested fixes.  Does asInstanceOf have a runtime cost?